docs/source/development/style-guide/versioning.rst

```diff
@@ -1,6 +1,11 @@
+.. _versioning:
```

Comment:
Don't these get added automatically now with the auto-section labeling extension?
https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/59b42d30453b769455e943ca9913a2cd1317f27a/docs/source/conf.py#L44

Reply:
Hmm, i'm not sure. I know that I needed to add these in order to get the docs to link properly. Without them, i get the warning `naming-conventions.rst:43: WARNING: undefined label: 'data-products-versioning'`

Reply:
Ah, thank you! Yes, that works.

These little gotchas drive me crazy 😅